Sources/MarathonCore/Create.swift

```diff
@@ -55,20 +55,55 @@ internal final class CreateTask: Task, Executable {
 
             return try editTask.execute()
         }
+      
+        let scriptFile = try createScriptIfNeeded(atPath: path)
+        
+        if arguments.contains("--tests") {
+            try createTests(atPath: path)
+        }
 
+        if !argumentsContainNoOpenFlag {
+            let script = try scriptManager.script(at: scriptFile.path, allowRemote: false)
+            try script.edit(arguments: arguments, open: true)
+        }
+    }
+    
+    private func createScriptIfNeeded(atPath path: String) throws -> File {
+        if arguments.contains("--tests"), let file = try? Folder.current.file(atPath: path) {
```

Comment:
Why is `arguments.contains("--tests")` a condition here? Don't we always want to return if the file already exists?